app/konnect/dev-portal/index.md

```diff
@@ -9,5 +9,13 @@ document and manage Application registration for your Services. Admin roles publ
 
 The Dev Portal lives at a separate URL from Konnect and requires that all users, including Admin roles, [register as a Developer](konnect/dev-portal/developers/dev-reg/). 
 
+## Customize the Dev Portal appearance
+
+The {{site.konnect_short_name}} Dev Portal can be customized via the Appearance settings. 
+
+To access the Appearance settings, click **Dev Portal** then **Appearance**.
+
+Here, you will be able to add a logo, select a pre-set theme, and modify background colors, text colors, button colors, and fonts. 
```

Comment:
Can you make this into a bulleted list and be more specific about the things that can be changed? The reason I ask for that is because on-prem Dev Portal and Cloud Dev Portal have different levels of customization available, and we've been asked in the past to make it clear what is possible and what isn't.